src/Stratis.Bitcoin/Configuration/Settings/ConnectionManagerSettings.cs

```diff
@@ -182,6 +180,10 @@ public static void BuildDefaultConfigurationFile(StringBuilder builder, Network
             builder.AppendLine("####ConnectionManager Settings####");
             builder.AppendLine($"#The default network port to connect to. Default { network.DefaultPort }.");
             builder.AppendLine($"#port={network.DefaultPort}");
+            builder.AppendLine($"#Accept connections from the outside.");
+            builder.AppendLine($"#listen=<0 or 1>");
+            builder.AppendLine($"#Forces the node to listen for incoming connections.");
```

Comment:
maybe add in braces `(this can be used to accept incoming connections when -connect is specified)`

Reply:
Fixed 👍